/**:
  ros__parameters:

    # topics:
    #   odom_topic: /orca/odom
    #   twist_topic: /dvl/twist
    #   pose_topic: /dvl/pose
    #   guidance_topic: /guidance/los
    #   thrust_topic: /thrust/wrench_input
    #   softwareoperation_topic: /softwareOperationMode
    #   killswitch_topic: /softwareKillSwitch

    LQR_params:
      q_surge: 75
      q_pitch: 175
      q_yaw: 175

      r_surge: 0.3
      r_pitch: 0.4
      r_yaw: 0.4

      i_surge: 0.3
      i_pitch: 0.4
      i_yaw: 0.3

      i_weight: 0.5

    dt: 0.1

    inertia_matrix: [30.0, 0.6, 0.0, 0.6, 1.629, 0.0, 0.0, 0.0, 1.729]
